app/helpers/adhoq/application_helper.rb

```diff
@@ -17,7 +17,7 @@ def schema_version
         ActiveRecord::SchemaMigration.maximum(:version)
       else
```

Comment:
> Looks like that defined?(ActiveRecord::SchemaMigration) condition is unnecessary.

 I understood that it can be deleted, if Adhoq don't support 3.2.
I'll remove this.